### DISCUSSION TITLE
Thumbnail Skeleton width bug

### DISCUSSION THREAD

**Discussion by guidobotta** [TiendaNube/nimbus-design-system#248] [Q&A] [answered]
I encountered a problem with `Thumbnail.Skeleton`, I have the following case:

```jsx
return <Thumbnail.Skeleton aspectRatio="2/1" width="100%" />;
```

Which translates to the following html

```html
<div class="nimbus-thumbnail_skeleton__131a6hn3">
  <div data-testid="thumbnail-skeleton-container" class="nimbus-thumbnail_aspectRatio_2/1__131a6hn9">
    <div class="nimbus-skeleton_base__17vaova1" style="--width__3tkju40: 100%; --height__3tkju41: 100%; --borderRadius__3tkju42: 0.5rem;">
    </div>
  </div>
</div>
```

The expected behavior for my case is the following:

<img width="1160" alt="Captura de pantalla 2024-08-13 a la(s) 10 50 00" src="https://github.com/user-attachments/assets/249e2ab6-dc51-468e-8ef0-8fd8bddeb023">

However, this is the actual result:

<img width="1162" alt="Captura de pantalla 2024-08-13 a la(s) 10 49 22" src="https://github.com/user-attachments/assets/48bc5709-f0e3-4f7e-81db-1ca43b4780b7">

The problem seems to be that the first two divs don't occupy the space provided by the parent node, so when the skeleton div gets a width of 100%, this 100% is actually 0px.

A possible solution would be to add `width: 100%` to the parents div, but I'm not sure if this is correct and enough.

**Comment by joacotornello** [comment, marked as answer]
Solved vía #289 in `@nimbus-ds/components@5.10.1`
